### PR TITLE
[BACKLOG-36973] - Page Zoom (AA) - Browse File perspective firefox issue

### PR DESCRIPTION
**Pull request by soagarwal1**
@pentaho/wcag Please review

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-870/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 9s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-870/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-870/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 9s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **398**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-870/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 1]
```diff
@@ -8968,6 +8993,12 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
   width:auto !important;
 }
 
+@supports (-moz-appearance:none) {
+  #fileBrowserFiles .body {
+    width:-moz-available !important;
+  }
+}
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:9000]: NOTE: Firefox select/listboxes grow to the size of the biggest option text.

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 2]
```diff
@@ -4515,17 +4515,42 @@ div.dijitDateTextBox.dijitComboBox {
...
 
+.fileBrowserColumn > .body {
+  display: flex;
+  flex-direction: column;
+  flex: none;
+}
+.fileBrowserColumn > .body > .folder {
+  width: max-content;
+  min-width: 100%;
+}
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:4526]: NOTE: These and following rules fix the selection rectangle of folder tree nodes not stretching till the end of line.

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 3]
```diff
@@ -4515,17 +4515,42 @@ div.dijitDateTextBox.dijitComboBox {
...
   font-family: opensansregular, Helvetica, Arial, Sans serif;
   position: relative;
+  display: flex;
+  flex-direction: row;
+}
+.fileBrowserColumn .body .folder .element > * {
+  flex: none;
+}
+.fileBrowserColumn .body .folder .element > .title {
+  flex: 1 0 auto;
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:4553]: NOTE: Allowing the title to grow allows _clicking_ anywhere on the line to select, even beyond the text.

**Pull request merged**